### PR TITLE
LPS-147394 Add support for Container HTML Tag

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	'header',
+	'nav',
+	'section',
+	'article',
+	'main',
+	'aside',
+	'footer',
+];
+
+export default function ContainerGeneralPanel({item}) {
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js:38]:
```suggestion
export default function ContainerAdvancedPanel({item}) {
```

😛

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			/>
+			<p className="small text-secondary">
+				{Liferay.Language.get('misusing-this-setup-might-impact-seo')}
+			</p>
+		</>
+	);
+}
+
+ContainerGeneralPanel.propTypes = {
+	item: getLayoutDataItemPropTypes({
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js:77]: Apart from the name that should be ContainderAdvancedPanel I think we should limit here the documentation to the things that this panel is using only

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js:38]: 😞

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/ContainerAdvancedPanel.js:77]: You are right 🥺

**Comment by lfbesada** [comment]
LGTM! Added a few commits in order to solve CI failures
See: https://github.com/liferay-echo/liferay-portal/pull/7418

**Pull request closed without merging**